.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
     # digraphs and other kernel extensions using C++ code (due to GCC's
     # `profiler_trace.h` calling `atexit`; but that fails in loadable modules
     # for technical reasons)
-    - env: TEST_SUITES=testpackages CFLAGS="-O2" LDFLAGS=
+    - env: TEST_SUITES=testpackages NO_COVERAGE=1 CFLAGS="-O2" LDFLAGS=
```

Comment:
I don't understand, what has this change to do with the description of the PR? And what is its motivation?

Reply:
If you just take the first commit, `testpackages` now fails because no coverage output is produced, and `MergeLineByLineProfiles` errors.

Two fixes are possible: Don't error even if the list of profiles to merge is empty (which means we also do not detect when there wasn't any profiling data produced erroneously), or explicitly don't expect coverage.

Or, if we do expect coverage for `testpackages`, then there is a bug in how these tests are run as they don't produce coverage.

Reply:
OK, so no I understand why `NO_COVERAGE` is used here. I'd greatly appreciate if the comment above this was extended to mention this, too (just like it explains about `--coverage` being dropped).

Reply:
Perhaps like this:
> Since no coverage data is produced anyway, set `NO_COVERAGE=1` to avoid calling the coverage submission code.